higher_health/tasks.py

```diff
@@ -0,0 +1,89 @@
+from typing import Text
+from urllib.parse import urljoin
+
+import requests
+from celery.exceptions import SoftTimeLimitExceeded
+from django.conf import settings
+from requests.exceptions import RequestException
+
+from config.celery import app
+from higher_health.models import Covid19Triage
+
+
+@app.task(
+    autoretry_for=(RequestException, SoftTimeLimitExceeded),
+    retry_backoff=True,
+    max_retries=15,
+    acks_late=True,
+    soft_time_limit=10,
+    time_limit=15,
+)
+def submit_healthcheck_to_eventstore(healthcheck_id: Text) -> None:
+    healthcheck = Covid19Triage.objects.select_related(
+        "facility_destination_university",
+        "facility_destination_campus",
+        "facility_destination_campus__university",
+    ).get(id=healthcheck_id)
+
+    data = {
+        "deduplication_id": str(healthcheck.id),
+        "msisdn": healthcheck.msisdn,
+        "first_name": healthcheck.first_name or "",
+        "last_name": healthcheck.last_name or "",
+        "source": healthcheck.source,
+        "province": healthcheck.province,
+        "city": healthcheck.city,
+        "age": healthcheck.age,
+        "date_of_birth": (
+            healthcheck.date_of_birth.isoformat() if healthcheck.date_of_birth else None
+        ),
+        "fever": healthcheck.fever,
+        "cough": healthcheck.cough,
+        "sore_throat": healthcheck.sore_throat,
+        "difficulty_breathing": healthcheck.difficulty_breathing,
+        "exposure": healthcheck.exposure,
+        # Our database field allows this to be `None`, but this is required on the API
+        # so if we have `None`, we send `False`. We'll store the raw value in the
+        # `data` field
+        "tracing": bool(healthcheck.confirm_accuracy),
+        "risk": healthcheck.risk,
+        "gender": healthcheck.gender,
+        "city_location": healthcheck.location,
+        "smell": healthcheck.smell,
+        "preexisting_condition": healthcheck.preexisting_condition,
+        "rooms_in_household": healthcheck.rooms_in_household,
+        "persons_in_household": healthcheck.persons_in_household,
+        "completed_timestamp": healthcheck.completed_timestamp.isoformat(),
+        "data": {
+            "confirm_accuracy": healthcheck.confirm_accuracy,
```

Comment:
Isn't this called `tracing` in the API?

Reply:
Yes it is, we have it as `tracing` above. This is just to store the original value, in case we converted `None` to `False`, and we want to know what the original value was.

I don't really have a string preference to using `tracing` or `confirm_accuracy` here, not sure if either is clearer when looking at the data without the context of this code.